docs/HowToGuides/GettingStarted.md

```diff
@@ -253,41 +253,30 @@ Phew, that's a lot to digest! Now let's proceed to the actual build itself!
 
 ### The actual build
 
-1. Build the toolchain with optimizations, debuginfo, and assertions, using
-   Ninja.
-   - macOS:
-     ```sh
-     utils/build-script --skip-build-benchmarks \
-       --skip-ios --skip-watchos --skip-tvos --swift-darwin-supported-archs "$(uname -m)" \
-       --sccache --release-debuginfo --swift-disable-dead-stripping
-     ```
-   - Linux:
-     ```sh
-     utils/build-script --release-debuginfo
-     ```
-     If you installed and want to use Sccache, include the `--sccache` option in
-     the invocation as well.
-   <!-- FIXME: Without this "hard" line break, the note doesn’t get properly spaced from the bullet -->
-   <br />
+Build the toolchain with optimizations, debuginfo, and assertions, using Ninja:
 
-   > **Note**  
-   > If you are planning to work on the compiler on macOS, but not the parts that are
-   > written in Swift, pass `--bootstrapping=hosttools` to speed up local
-   > development.
+- macOS:
+  ```sh
+  utils/build-script --skip-build-benchmarks \
+    --skip-ios --skip-watchos --skip-tvos --swift-darwin-supported-archs "$(uname -m)" \
+    --sccache --release-debuginfo --swift-disable-dead-stripping \
+    --bootstrapping=hosttools
+  ```
+- Linux:
+  ```sh
+  utils/build-script --release-debuginfo
+  ```
+  - If you want to additionally build the Swift core libraries, i.e.,
+    swift-corelibs-libdispatch, swift-corelibs-foundation, and
+    swift-corelibs-xctest, add `--xctest` to the invocation.
 
-   This will create a directory `swift-project/build/Ninja-RelWithDebInfoAssert`
-   containing the Swift compiler and standard library and clang/LLVM build artifacts.
-   If the build fails, see [Troubleshooting build issues](#troubleshooting-build-issues).
+- If you installed and want to use Sccache, add `--sccache` to the invocation.
```

Comment:
There is a section with installation instructions for sccache on development tips I guess, should we link this to that reference?

Reply:
We have installation instructions earlier in the file too:

https://github.com/apple/swift/blob/24814c5e9da1552e3886bf380c53da1f10d8335a/docs/HowToGuides/GettingStarted.md?plain=1#L142-L150